cfgov/unprocessed/js/routes/on-demand/feedback-form.js

```diff
@@ -4,39 +4,60 @@
 
 'use strict';
 
+var messages = {
```

Comment:
I moved the error messaging into the config file and did a bit of cleanup for now, but I agree that it would make sense for the logic to go somewhere else. I'll look at that for a future PR.